README.md

```diff
@@ -94,9 +94,22 @@ For a similar reason, this polyfill can't proxy `Array` objects very well - but
 
 # Usage
 
+## To assign Proxy to the global object:
 Include the JavaScript at the start of your page, as an ES6 module (although browsers that support ES6 modules support `Proxy` natively) or include it as a dependency to your build steps.
 The source is in ES6, but the included, minified version is ES5.
 
+## To consume Proxy as a function
+Require from your app the file `./src/proxy.js`, which exports a proxy polyfill function.
+```js
+// In ES6/ES2015 supporting environments
```

Comment:
I'm confused. This doesn't work in browsers like Chrome or Safari, which have no idea how to deal with commonJS. I get that Node/ESM/whatever might, but...

It's fine for now, I'll update the comment. :)